lib/rspec/core/formatters/minimal_formatter.rb

```diff
@@ -0,0 +1,18 @@
+RSpec::Support.require_rspec_core "formatters/base_formatter"
+
+module RSpec
+  module Core
+    module Formatters
+      # @private
+      class MinimalFormatter < BaseFormatter
+        Formatters.register self, :example_failed, :dump_profile
+
+        def example_failed(failure)
+          output.puts "#{failure.example.location}:#{failure.example.description}"
+        end
+
+        def dump_profile(_profile); end
```

Comment:
Ok, so if you want to silence output, can we get a comment here explaining that? Also what about the other output? start / end etc?

Reply:
I initially chose to discard the profile output because it produce a big chunk of information which is not really useful in the context of an editor (and since the formatter was renamed "minimal" it may make even more sense to discard any "extra" output), but I do not have strong opinion about this, and if you prefer to keep these messages I can amend my work accordingly.

I found out that `message` can be defined to discard more informational messages, and added a commit to include this with the requested comment.  Please review and comment / resolve this conversation.

Thanks!